static/src/javascripts/projects/commercial/modules/brazeBanner.js

```diff
@@ -140,6 +141,9 @@ const getMessageFromBraze = async (apiKey: string, brazeUuid: string): Promise<b
 };
 
 const canShow = async (): Promise<boolean> => {
+    const canShowTiming = measureTiming('braze-banner');
```

Comment:
Super minor naming thing but I wonder if `timing` would work better as a name here (we're already in `canShow` so do we need to repeat `canShow` in the const name?)

Reply:
Good idea.